app-config.yaml

```diff
@@ -43,6 +43,16 @@ integrations:
     #   apiBaseUrl: https://ghe.example.net/api/v3
     #   token: ${GHE_TOKEN}
 
+kafkaInfo:
+  # Metric we'll use for tracking topic use and lag
+  lagMetric: 'kafka_consumergroup_group_topic_sum_lag'
```

Comment:
Should this be an env var? If someone in the community or in a different BU or a different Kafka provider used this plugin would it work with this hard coded?